apps/coscrad-frontend/src/emotion-components/theme.ts

```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled';
+import { createTheme } from '@mui/material';
+import { primaryColor, secondaryColor } from './colors';
+
+export const theme = createTheme({
```

Comment:
Presumably we inject a config (maybe reuse the content config for now?) that specifies a palette number (e.g. 100)? Ideally the config would specify the palette itself. But a hybrid combo might be nice. Users can use an officially comissioned theme or even submit their configured theme as an official option. I'd like to hear thoughts from the team on this. Having one approach will be less maintenance and risk.